server/server.go

```diff
@@ -102,6 +108,12 @@ func (ts *Server) OnBoot(_ gnet.Engine) gnet.Action {
 			time.Sleep(100 * time.Millisecond)
 		}
 	}()
+	go func() {
+		for {
+			time.Sleep(60 * time.Second)
+			ts.raft.Snapshot()
+		}
+	}()
```

Comment:
You don't need to run snapshots periodically, raft will do it by it self to compact logs. See [here](https://github.com/hashicorp/raft/blob/2b0032e390b888b0058d221989886d090b9d9609/api.go#L615-L615)

You can implement a snapshot command though that would take a user snapshot and backup the DB.

Reply:
Oh got it.

Reply:
I see that function is there, but when I run the server - I don't see snapshot creating.